### DISCUSSION TITLE
fzf preview tab not working

### DISCUSSION THREAD

**Discussion by DrSensor** [leo-arch/clifm#172] [Q&A] [answered]
Hi, anyone know why it's not working?

![2022-11-22_03-10](https://user-images.githubusercontent.com/4953069/203149349-0ff259f2-798e-4158-bd5b-2f3303654c8c.png)

`clifm --preview deno.json` works just fine.

My guess is due to `;` prefix but I don't know where it came from. In v1.17 it works fine (I'm using BFG) but somehow in the new release the BFG plugins disabled 🤔 (then I configure `FzfPreview=true` manually)

I'm in Arch with clifm v1.8-Otis

**Comment by leo-arch** [comment, marked as answer]
Hi @DrSensor. Thanks for reporting.

The issues you describe might be already solved upstream (a [similar issue](#159) was solved indeed shortly after the release of 1.8). Have you tried running the latest git version?

```sh
mkdir build
cd build
git clone https://github.com/leo-arch/clifm
cd clifm
make -f misc/GNU/Makefile _NERD=1 install
```
This installs **clifm** 1.8.7 (with Nerdfonts support).

Give it a try and let me know if it works for you.

**Reply by DrSensor** [reply]
alright it works 👍

**Reply by leo-arch** [reply]
Cool!
